backend/persistent/datastores/Datastores.js

```diff
@@ -27,6 +29,12 @@ import * as ApacheImpala from './impala';
 
 function getDatastoreClient(connection) {
     const {dialect} = connection;
+
+    // handle test mode:
+    if (getSetting('TEST_MODE')) {
```

Comment:
can this be done without introducing a new ENV var?

Reply:
I am not sure of the other way. We would typically need to enable this from outside the container, ie. when we run the container. 

```
PLOTLY_CONNECTOR_TEST_MODE=true docker run -p 9494:9494 falcon
```

Please let me know if you know of any other way to accomplish the same !!